Meshery-Adapters/traefik-meshery-adapter/index.json

```diff
@@ -0,0 +1,47 @@
+{
+    "title": "Working with Meshery and Traefik Mesh",
+    "description": "Meshing around with the Meshery adapter for Traefik Mesh",
+    "difficulty": "Beginner",
+    "time": "10 minutes",
+    "details": {
+      "steps": [
+        {
+          "title": "Let's get meshing!",
+          "text": "step1.md",
+          "courseData": "step1-background.sh",
+          "code": "step1-foreground.sh"
+        },
+        {
+          "title": "Deploy Traefik Mesh",
+          "text": "step2.md"
+        },
+        {
+          "title": "Run a sample app",
+          "text": "step3.md"
+        }
+      ],
+      "intro": {
+        "text": "intro.md"
+      },
+      "finish": {
+        "text": "finish.md"
+      },
+      "assets": {
+        "host01": [
+            {"file": "auth.json", "target": "/root", "chmod": "+x"},
+            {"file": "shell.sh", "target": "/root", "chmod": "+x"}
+        ]
+      }
+    },
+    "environment": {
+      "uieditorpath": "/root/example",
+      "hideHiddenFiles": true,
+      "exclusionPatterns": ["*test", "./test/*", "./logs"],
+      "showdashboard": true,
+      "dashboards": [{"name": "Meshery Server", "port": 9081},{"name": "Emojivoto", "port": 8080}],
+      "uilayout": "terminal-iframe"
+    },
+    "backend": {
+      "imageid": "kubernetes-cluster-running"
+    }
+  }
```

Comment:
@warunicorn19  have you tested `kubernetes-cluster-running` image-id?

Reply:
Haven't tested it, but I thought that's what we required after yesterday's call .

Reply:
Yep we do require Kubernetes running, we have to make sure that it compatible with existing instructions.

Reply:
@warunicorn19  what's your conclusion about `kubernetes-cluster-running` image?